app/src/androidTest/assets/lifiswap.json

```diff
@@ -137,6 +137,6 @@
       "vault_public_key_ecdsa": "0342d6eb3e536bd1d6f57a8388afb09936aa64160c5e2ebee76d791b4844a06770",
       "lib_type": "DKLS"
     },
-    "expected_image_hash": ["4324862d2c76549bdf6578d070258f1c6c8bcbad27ab7b3dc8ca8af8452fb125"]
+    "expected_image_hash": ["63fbeb1e3fd2e6d5c806bb1b2e9c8b4f56c79f70257de3c7e799553d0fb99f97"]
```

Comment:
@JaimeToca , this is going to cause cross device signing to fail ,  we should not change this
1. The initial device should make sure the `gas_price` is correct and useable
2. The pair device should strictly respect the `gas_price` field in the payload